devtools-ci-index.yaml

```diff
@@ -282,7 +282,7 @@
           status-context: "ci.centos.org PR build (fabric8-planner)"
           org-list:
             - fabric8-planner
-          success-comment: "$ghprbPullAuthorLoginMention Your image is available in the registry: Run `docker run -e PROXY_PASS_URL=\"https://api.free-stg.openshift.com\" -p 8080:8080 {image_name}`."
+          success-comment: "$ghprbPullAuthorLoginMention Your image is available in the registry: Run `docker run -e PROXY_PASS_URL=\"https://api.free-stg.openshift.com\" -p 8080:8080 registry.devshift.net/{image_name}:SNAPSHOT-PR-$ghprbPullId`."
```

Comment:
why are you using a proxy here ?

Reply:
I hope this helps https://github.com/openshiftio/openshift.io/issues/1639#issuecomment-351342439